warehouse/integrations/azure-synapse/azure-synapse.go

```diff
@@ -135,7 +149,7 @@ func connect(cred credentialsT) (*sql.DB, error) {
 		Host:     net.JoinHostPort(cred.host, strconv.Itoa(port)),
 		RawQuery: query.Encode(),
 	}
-	pkgLogger.Debugf("synapse connection string : %s", connUrl.String())
```

Comment:
This contains sensitive information like passwords and accessKeys. We should probably not log it.